### PR TITLE
Fixes #60: Feature-Request-Add-Fake-Data

### LINKED ISSUE #60: [Feature Request]: Add Fake Data
### Is your feature request related to a problem? Please describe.

We need fake data to serve to our frontend via APIs

### Describe the solution you'd like

Add fake data for following things using Faker gem.

Note: If you don't find fake data go to https://www.practo.com/ register as doctor and then based on that add fake data.

- [ ] Add degree - For this you can search top 10 medical degrees in India and use those names for now.
- [ ] Add institutes - Search for top 20 institutes in India and add them in database via seeds.rb file
- [ ] Add registration councils - Search for top 10 registration counsils.
- [ ] Add specializations - Search doctor related specialisations atleast 10

### Describe alternatives you've considered

_No response_

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by KvngJamesNG**
Creates seeds for:
doctor_profile
patient_profile 
And also their relations

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -37,3 +37,6 @@
 .DS_Store
 /.idea/*.xml
 /.idea/KlinicCon-Backend.iml
+
+# Ignore credential for decrypting credentials and more.
+/config/credentials.yml.enc
```
Comment (rt4914) [inline .gitignore:42]: Undo changes from this file. It is not part of this PR.

File: Gemfile.lock [hunk 2]
```diff
@@ -281,14 +281,15 @@ GEM
...
+    webrick (1.8.2)
     websocket-driver (0.7.6)
       websocket-extensions (>= 0.1.0)
     websocket-extensions (0.1.5)
-    zeitwerk (2.6.17)
+    zeitwerk (2.7.1)
 
 PLATFORMS
   x86_64-darwin-19
+  x86_64-darwin-21
```
Comment (rt4914) [inline Gemfile.lock:292]: Undo gemfile.lock related changes too. They should not be part of this PR.

File: app/models/doctor_profile.rb [hunk 3]
```diff
@@ -4,3 +4,4 @@ class DoctorProfile < ApplicationRecord
   belongs_to :institute
   has_one :user
 end
+
```
Comment (rt4914) [inline app/models/doctor_profile.rb:7]: Remove this extra blank line.

File: config/initializers/money.rb [hunk 4]
```diff
@@ -82,7 +82,7 @@
 
   # If you would like to use I18n localization (formatting depends on the
   # locale):
-  # config.locale_backend = :i18n
+  config.locale_backend = :i18n
```
Comment (rt4914) [inline config/initializers/money.rb:85]: This change is not required for this PR.

File: db/seeds.rb [hunk 5]
```diff
@@ -1,9 +1,162 @@
...
+# Create establishments and associate with address
+establishment = Establishment.create!(
+  name: Faker::Company.name,
+  address_id: address.id,   # associate with address using address_id
+  latitude: Faker::Address.latitude,
+  longitude: Faker::Address.longitude,
+  maps_location: "https://goo.gl/maps/#{Faker::Address.street_address.gsub(' ', '-')}"  # Example for a map location
+)
+
+# Create doctor profiles
```
Comment (rt4914) [inline db/seeds.rb:97]: Couple of suggestions for fake data.

This code creates only 1 doctor_profile, it should be atleast 20.
This code creates only 1 patient_profile, it should be atleast 50.

This code adds only 1 review it should be atleast 20-50
This code adds only 1 service it should be atleast 20-50

**Review by rt4914 — CHANGES_REQUESTED** [review]
Good overall but requested chnges.

File: .gitignore (hunk 1 above)
Comment (KvngJamesNG) [reply .gitignore:42]: done.

File: Gemfile.lock (hunk 2 above)
Comment (KvngJamesNG) [reply Gemfile.lock:292]: done.

File: app/models/doctor_profile.rb (hunk 3 above)
Comment (KvngJamesNG) [reply app/models/doctor_profile.rb:7]: done.

File: config/initializers/money.rb (hunk 4 above)
Comment (KvngJamesNG) [reply config/initializers/money.rb:85]: done.

File: db/seeds.rb (hunk 5 above)
Comment (KvngJamesNG) [reply db/seeds.rb]: I think done also, 


Ignore my comment on the code I will take them off soon. 😂
